python/dgl/sparse/elementwise_op.py

```diff
@@ -2,43 +2,26 @@
 """DGL elementwise operator module."""
 from typing import Union
 
-from .diag_matrix import DiagMatrix
 from .sparse_matrix import SparseMatrix
 from .utils import Scalar
 
 __all__ = ["add", "sub", "mul", "div", "power"]
 
 
-def add(
-    A: Union[DiagMatrix, SparseMatrix], B: Union[DiagMatrix, SparseMatrix]
-) -> Union[DiagMatrix, SparseMatrix]:
-    r"""Elementwise addition for ``DiagMatrix`` and ``SparseMatrix``, equivalent
-    to ``A + B``.
-
-    The supported combinations are shown as follows.
-
-    +--------------+------------+--------------+--------+
-    |    A \\ B    | DiagMatrix | SparseMatrix | scalar |
-    +--------------+------------+--------------+--------+
-    |  DiagMatrix  |     ✅     |      ✅      |   🚫   |
-    +--------------+------------+--------------+--------+
-    | SparseMatrix |     ✅     |      ✅      |   🚫   |
-    +--------------+------------+--------------+--------+
-    |    scalar    |     🚫     |      🚫      |   🚫   |
-    +--------------+------------+--------------+--------+
```

Comment:
Shall we keep the table to show that adding a sparse matrix with a scalar is not allowed?

Reply:
I removed the tables because they are too simple. They are 2x2 and mostly with checkmarks.